proxmove

```diff
@@ -1983,12 +1986,25 @@ class VmMover(object):
                 dst_vm = self.dst_pve.get_vm(
                     src_vm.basename, suffix=SUFFIX_CREATING)
             except ProxmoxVm.DoesNotExist:
+                dst_vm = None
+
+            if not dst_vm:
                 # Pristine VMs: start, by translating config and
                 # creating a new VM.
                 dst_config = translator.config(src_vm.get_config())
+
+                if self.opt_keep_id:
+                    dst_id = src_vm.id
+                    if not self.dst_pve.check_vmid_free(dst_id):
+                        raise PrepareError(
```

Comment:
Too late to raise PrepareErrors here. This should've been checked in `move_vm()` (also done in dry-run).

Here we can safely assume the ID is free and let Proxmox handle duplicate VM id errors.